src/lib/apify_command.js

```diff
@@ -2,18 +2,31 @@ const { Command } = require('@oclif/command');
 const { finished } = require('stream');
 const { promisify } = require('util');
 const { argsToCamelCase } = require('./utils');
+const { mixpanel, getOrCreateLocalDistinctId, isTelemetryEnabled } = require('./telemetry');
+const { detectInstallationType } = require('./version_check');
 
 /**
  * Adding parsing flags to oclif Command class
  */
 class ApifyCommand extends Command {
     parse(cmd) {
         const { flags, args } = super.parse(cmd);
+        let distinctId = null;
         const parsedFlags = argsToCamelCase(flags);
-        return {
-            flags: parsedFlags,
-            args,
-        };
+
+        if (isTelemetryEnabled) {
```

Comment:
I'd consider placing this into the try/catch to make sure that error in tracking will never affect a usage